spec/rubocop/target_finder_spec.rb

```diff
@@ -440,6 +440,19 @@
       expect(found_basenames).to include('ruby3.rb')
       expect(found_basenames).to include('ruby4.rb')
     end
+
+    unless RuboCop::Platform.windows?
```

Comment:
Thank you for investigation. I think it useful to have comment and address link on why the Windows test is skipped.

Reply:
OK! I added.